src/tests/test_config.py

```diff
@@ -34,3 +34,9 @@ def test_safeGetInt(self):
             BMConfigParser().safeGetInt('nonexistent', 'nonexistent'), 0)
         self.assertEqual(
             BMConfigParser().safeGetInt('nonexistent', 'nonexistent', 42), 42)
+
+    def test_reset(self):
```

Comment:
I don't think this is a correct test. The correct test should
1. load a test config file
2. either call `_reset` or load an empty file
3. the assert probably can remain as it is, but I would maybe put another assert in front of it, that tests if the `settingsversion` section is absent